### PR TITLE
Adding helper method to retrieve region code

### PR DESCRIPTION
**Pull request by nanoscott**

### CODE REVIEW

File: src/foam/nanos/auth/Address.js [hunk 1]
```diff
@@ -1063,6 +1063,22 @@ foam.CLASS({
...
+    },
+    {
+      name: 'retrieveRegionCode',
+      type: 'String',
+      args: [ { name: 'x', type: 'Context' } ],
+      javaCode: `
+        String regionCode = "";
+        Region region = findRegionId(x);
+        if ( region != null ) {
+          regoinCode = region.getRegionCode();
```
Comment (jlhughes) [inline src/foam/nanos/auth/Address.js]:
```suggestion
          regionCode = region.getRegionCode();
```

**Comment by jlhughes** [comment]
@nanoscott How will this be used?

**Comment by nanoscott** [comment]
@jlhughes  .. here's a how i'm planning on using it: https://github.com/nanoPayinc/NANOPAY/pull/12901

**Review by lchanmann — COMMENTED** [review]
LGTM

**Review by jlhughes — APPROVED** [review]

**Pull request merged**